src/database/connection.ts

```diff
@@ -47,6 +47,7 @@ export const tableNames = {
   MONEY_TRANSACTIONS: 'moneytransactions',
   LOANS: 'loans',
   PRODUCT_PAYMENTS: 'productpayments',
+  MEMOS: 'memos',
```

Comment:
is this memos change supposed to be here?

Reply:
I think i branched off the wrong branch :P